WPF-DataBinding/readme.md

```diff
@@ -0,0 +1,210 @@
+# WPF Databinding
+
+
+## Inhaltsverzeichnis
+1. Definition
+2. Basic data binding concepts
+3. Richtung des data flows
+4. Synchronisation
+5. Data binding XAML
+6. Data binding code behind
+7. Daten konvertieren
+
+## Definition
+
+Data binding ist der Prozess, der die Verbindung zwischen einem UI-Element und den Daten die diese anzeigt herstellt.
+Wenn Daten des Data Binding ge�ndert werden, werden diese auch automatisch im UI-Element aktualisiert. Hierbei k�nnen die Daten auch von au�erhalb in der UI durch den
+Benutzer ver�ndert werden und diese werden dann in der Datenbank aktualisiert. 
```

Comment:
Vorsicht: durch das DataBinding werden die Daten nicht automatisch in der DB geändert!
Vielmehr wird ein Property im ViewModel geändert, welches dann beim Anstoss des Speicherns dann durch den Code in der DB gespeichert wird.